pkg/universe.dagger.io/go/image.cue

```diff
@@ -12,6 +12,8 @@ _#DefaultVersion: "1.18"
 	version: *_#DefaultVersion | string
 
 	packages: [pkgName=string]: version: string | *""
+    // FIXME Remove once golang image include 1.18 *or* go compiler is smart with -buildvcs
```

Comment:
Added the FIXME 🙃

Reply:
Yes, keep this, but it's `Remove once golang image includes git (docker-library/golang#415) **or** go compiler is smart with -buildvcs (golang/go#51748)`.